include/data_management/data_source/csv_feature_manager.h

```diff
@@ -451,8 +451,8 @@ class CSVFeatureManager : public StringRowFeatureManagerIface
      *  \param[out] rowBuffer    Pointer to a Buffer View to store the result of parsing
      *  \param[in]  ntRowIndex   Position in the Numeric Table at which to store the result of parsing
      */
-    virtual void parseRowIn(char * rawRowData, size_t rawDataSize, DataSourceDictionary * dictionary,
-                            services::BufferView<DAAL_DATA_TYPE> & rowBuffer, size_t ntRowIndex) DAAL_C11_OVERRIDE
+    virtual void parseRowIn(char * rawRowData, size_t rawDataSize, DataSourceDictionary * /*dictionary*/,
+                            services::BufferView<DAAL_DATA_TYPE> & rowBuffer, size_t /*ntRowIndex*/) DAAL_C11_OVERRIDE
     {
         DAAL_ASSERT(dictionary);
```

Comment:
This will not work in debug mode (when we activate _asserts_).

Reply:
fixed. please review